cmd/agent/dist/conf.d/disk.d/conf.yaml.default

```diff
@@ -1,53 +1,162 @@
+    ## Note: On some linux distributions, rootfs is found and tagged as a device.
+    ## Add rootfs here to exclude.
+    #
     # excluded_filesystems:
     #   - tmpfs
 
-    # The (optional) excluded_disks parameter will instruct the check to
-    # ignore this list of disks.
-    # Linux example:
+    ## @param excluded_disks - list of strings - optional
+    ## The `excluded_disks` parameter instructs the check to
+    ## ignore this list of disks.
+    #
     # excluded_disks:
     #   - /dev/sda1
     #   - /dev/sda2
+
+    ## @param excluded_disk_re - string - optional
+    ## The `excluded_disk_re` parameter instructs the check to
+    ## ignore all disks matching this regex.
     #
-    # The (optional) excluded_disk_re parameter will instruct the check to
-    # ignore all disks matching this regex.
     # excluded_disk_re: /dev/sde.*
+
+    ## @param excluded_mountpoint_re - string - optional
+    ## The `excluded_mountpoint_re` parameter instructs the check to
+    ## ignore all mountpoints matching this regex.
     #
-    # The (optional) tag_by_filesystem parameter will instruct the check to
-    # tag all disks with their filesystem (for ex: filesystem:nfs).
-    # tag_by_filesystem: false
-    #
-    # The (optional) excluded_mountpoint_re parameter will instruct the check to
-    # ignore all mountpoints matching this regex.
-    # excluded_mountpoint_re: /mnt/somebody-elses-problem.*
+    # excluded_mountpoint_re: <MOUNT_POINT_REGEX>
+
+    ## @param all_partitions - boolean - optional - default: false
+    ## Instruct the check to collect from partitions even without device names.
+    ## Setting `use_mount` to true is strongly recommended in this case.
     #
-    # The (optional) all_partitions parameter will instruct the check to
-    # get metrics for all partitions. use_mount should be set to yes (to avoid
-    # collecting empty device names) when using this option.
     # all_partitions: false
+
+    ## @param tags - list of key:value strings - optional
+    ## List of tags to attach to every metric, event, and service check emitted by this integration.
+    ##
+    ## Learn more about tagging at https://docs.datadoghq.com/tagging
     #
-    # The (optional) device_tag_re parameter will instruct the check to apply additional tags to
-    # volumes/mountpoints matching these regexes. Multiple comma-separated tags are supported.
-    # You must properly quote the string if the pattern contains special characters e.g. colons.
-    # Character casing is ignored on Windows.
-    # device_tag_re:
-    #  /san/.*: device_type:san
-    #  /dev/sda3: role:db,disk_size:large
-    #  "c:": volume:boot
-    #
-    # The (optional) tags parameter allows you to customize tags for the instance
     # tags:
-    #   - optional_tag1
-    #   - optional_tag2
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
+
+    ## @param file_system_whitelist - list of regex - optional
```

Comment:
This parameter is only available in the Python version of the disk check. The Go version is more limited.
The Python version has it's own default configuration https://github.com/DataDog/integrations-core/blob/master/disk/datadog_checks/disk/data/conf.yaml.default.

The default configuration from integration is shipped by default. I think this file should only reflect the option available in the Go version of the check (with a disclaimer at the beginning maybe ?).

Reply:
Many option here are still only available in the python version:

- `file_system_blacklist`
- `device_whitelist`
- `mount_point_whitelist`
...